crates/ruff_linter/resources/test/fixtures/pylint/redefined_slots_in_subclass.py

```diff
@@ -0,0 +1,6 @@
+class Base:
+    __slots__ = ("a", "b")
+
+
+class Subclass(Base):
+    __slots__ = ("a", "d")  # [redefined-slots-in-subclass]
```

Comment:
Can you add some more examples for this? Like what would happen if there's another class in the hierarchy?

```
class Grandparent:
    __slots__ = ("a", "b")


class Parent(Grandparent):
    pass


class Child(Parent):
    __slots__ = ("c", "a")
```

Pylint does trigger for the above code but I don't think Ruff would do the same.